### PR TITLE
October newsletter

### PR DESCRIPTION
**Pull request by meenaxc**

### CODE REVIEW

File: content/roadmap/leightweight-renderers/_index.md
Comment (ramyaragupathy) [inline content/roadmap/leightweight-renderers/_index.md:1]: @meenaxc - what is this content about? Why is this included in this PR?

**Comment by HarelM** [comment]
It is advised to publish the Octrober newsletter before the end of October (or any other month for that matter)...
If there is a problem achieving this, please reach out and we'll help.

**Comment by meenaxc** [comment]
Noted. I was pausing on the TSC dates. Should we add: 

- developer office hours with the board sign-up 
- 30-day map challenge contest or sweepstakes?
